server/graphql/invoice_line/src/mutations/outbound_shipment_line/line/update.rs

```diff
@@ -607,9 +592,6 @@ mod test {
             "stockLineId": "stock_line_id input",
             "numberOfPacks": 1.0,
             "totalBeforeTax": 1.0,
-            "tax": {
-              "percentage": 1.0
-            }
           },
```

Comment:
Can't you just leave it in the api? we just don't currently use it?

Reply:
[e4883bd](https://github.com/openmsupply/open-msupply/pull/919/commits/e4883bddb6776c95933c6694b75a1071b2ba5783)